### DISCUSSION TITLE
Unable to verify code

### DISCUSSION THREAD

**Discussion by coasty489** [kachurovskiy/megaels#27] [General]
Hi Maxim. I’ve built your ELS and have been using it, but recently had to update the threading and cones section.
Now when I try to verify the code it has an error saying Spindle was not declared in this scope. (Line 704).
Are you able to suggest what has gone wrong.
I look forward to using my minilathe again.
Many thanks Mike.

**Comment by kachurovskiy** [comment]
Hi @coasty489, to see what's wrong please share your new code e.g. via https://gist.github.com/ or just check the line you changed that compiler is complaining about, ensure that variables you're changing are in scope (declared above where you're using them and not inside curly braces block you already exited).

chatgpt.com is also helpful sometimes if you just paste into in the error message.

**Comment by coasty489** [comment]
Hi Maxim.
Thanks for your time and reply. Your help has sorted it out and I have uploaded the revised code to the Arduino. Tomorrow I will see if it works.
Thanks again Mike.
